### PR TITLE
Prop for disabling touch events

### PR DESCRIPTION
**Pull request by zlangbert**
I'd like to be able to stop the user from navigating slides if needed

### CODE REVIEW

**Comment by oliviertassinari** [comment]
I'm up for this change :+1:, however, I have some feedbacks.

File: src/index.js [hunk 1]
```diff
@@ -49,6 +49,11 @@ const SwipeableViews = React.createClass({
      * If the computed speed is above this value, the index change.
      */
     threshold: React.PropTypes.number,
+
+      /**
```
Comment (oliviertassinari) [inline src/index.js]: I'm gonna add travis and an eslint check.
I suspect a wrong indent here.

File: src/index.js [hunk 2]
```diff
@@ -49,6 +49,11 @@ const SwipeableViews = React.createClass({
...
+       * If true disable touch events
+       */
+    disabled: React.PropTypes.bool
```
Comment (oliviertassinari) [inline src/index.js]: Could you add it to the Readme too ?

File: src/index.js [hunk 3]
```diff
@@ -94,6 +104,9 @@ const SwipeableViews = React.createClass({
     this.isScroll = undefined;
   },
   handleTouchMove(event) {
+    if (this.props.disabled)
```
Comment (oliviertassinari) [inline src/index.js]: This extra check is not good for perfs.
I think that I would be better
To not listen to the event.

File: src/index.js (hunk 1 above)
Comment (oliviertassinari) [inline src/index.js]: https://github.com/oliviertassinari/react-swipeable-views/blob/master/.eslintrc

**Comment by zlangbert** [comment]
Updated

File: src/index.js [hunk 4]
```diff
@@ -205,9 +212,14 @@ const SwipeableViews = React.createClass({
       translate: spring(index * 100, springConfig),
     };
 
+    const touchEvents = !this.props.disabled ? {
+      onTouchStart: this.handleTouchStart,
```
Comment (oliviertassinari) [inline src/index.js]: Can we use the spread to get the disabled property?
I plan to pass unused properties to the root div, so I need it.
I also think that negation are harder to reason about.

**Comment by oliviertassinari** [comment]
Still miss the update of the Readme to be perfect. If you don't have time to do it, I will merge this PR.

File: src/index.js (hunk 4 above)
Comment (oliviertassinari) [inline src/index.js]: (could be useful to add a className for instance)

**Comment by zlangbert** [comment]
Sorry, forgot. Added to readme

**Comment by zlangbert** [comment]
Changed the disabled code. Let me know if that's not what you meant

**Comment by oliviertassinari** [comment]
Thanks!

**Pull request merged**